plugins/GSdx/GSSetting.cpp

```diff
@@ -121,6 +121,11 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Enables external shader for additional post-processing effects.";
 		case IDC_FXAA:
 			return "Enables fast approximate anti-aliasing. Small performance impact.";
+		case IDC_AUTO_FLUSH:
+			return "Force a primitive flush when framebuffer is also an input texture. Warning: it is very costly on the performances. Help to fix some processing effects such as Jak shadows, GTA radiosity..."
```

Comment:
``Help to fix some processing effects such as Jak shadows, GTA radiosity...`` **->** ``Helps to fix post-processing effects on games such as Jak and GTA: San Andreas.``

**Note**: Could be improved even further.